src/raft.rs

```diff
@@ -740,7 +740,15 @@ impl<T: Storage> Raft<T> {
     /// changed (in which case the caller should call `r.bcast_append`).
     pub fn maybe_commit(&mut self) -> bool {
         let mci = self.mut_prs().maximal_committed_index().0;
-        self.raft_log.maybe_commit(mci, self.term)
+        let ok = self.raft_log.maybe_commit(mci, self.term);
+        let self_id = self.id;
+        self.update_committed(self_id);
```

Comment:
Why use `self.id` as a `committed_index`?

Reply:
I am trying to pass `self.raft_log.committed` to progress set for updating committed index